### PR TITLE
remove min,max and last hour(0 - 30)

### PR DESCRIPTION
**Pull request by rajnikantsh**

### CODE REVIEW

File: src/main/java/beam/utils/BeamCalcLinkStats.java [hunk 1]
```diff
@@ -55,7 +54,7 @@
     public BeamCalcLinkStats(final Network network, final TravelTimeCalculatorConfigGroup ttConfigGroup) {
         this.network = network;
         this.linkData = new ConcurrentHashMap<>();
-        this.nofHours = (int)TimeUnit.SECONDS.toHours(ttConfigGroup.getMaxTime());
+        this.nofHours = (int) TimeUnit.SECONDS.toHours(ttConfigGroup.getMaxTime());
```
Comment (carloscaldas) [inline src/main/java/beam/utils/BeamCalcLinkStats.java]: remove `this` qualifier. It is recommended only when there is name colision

File: src/main/java/beam/utils/BeamCalcLinkStats.java [hunk 2]
```diff
@@ -145,74 +138,53 @@ public void writeFile(final String filename) {
...
+                    } else {
+                        double ttsum = data.ttimes[SUM][i];
+                        if (ttsum == 0) {
+                            writeCommaAndStr(out, ttimesMin);
+                        } else {
+                            writeCommaAndStr(out, Double.toString(ttsum / data.volumes[SUM][i]));
+                        }
                     }
+
+                    out.write("\n");
```
Comment (carloscaldas) [inline src/main/java/beam/utils/BeamCalcLinkStats.java]: extract a class level constant to this

File: src/main/java/beam/utils/BeamCalcLinkStats.java [hunk 3]
```diff
@@ -145,74 +138,53 @@ public void writeFile(final String filename) {
...
 
-                        writeCommaAndStr(out, Double.toString(link.getFreespeed()));
+                    writeCommaAndStr(out, Double.toString(link.getFreespeed()));
 
-                        writeCommaAndStr(out, Double.toString(link.getCapacity()));
+                    writeCommaAndStr(out, Double.toString(link.getCapacity()));
 
-                        //WRITE STAT_TYPE
-                        writeCommaAndStr(out, statType[j]);
+                    //WRITE STAT_TYPE
```
Comment (carloscaldas) [inline src/main/java/beam/utils/BeamCalcLinkStats.java]: this comment is redundant. please remove it

**Review by carloscaldas — APPROVED** [review]
Please check comments before merging

**Comment by rajnikantsh** [comment]
this PR changes is merge into master via saperate PR(https://github.com/LBNL-UCB-STI/beam-utilities/pull/7). hence closing this

**Pull request closed without merging**